pl_examples/basic_examples/simple_image_classifier.py

```diff
@@ -66,47 +76,19 @@ def test_step(self, batch, batch_idx):
     def configure_optimizers(self):
         return torch.optim.Adam(self.parameters(), lr=self.hparams.learning_rate)
 
-    @staticmethod
-    def add_model_specific_args(parent_parser):
-        parser = parent_parser.add_argument_group("LitClassifier")
-        parser.add_argument('--hidden_dim', type=int, default=128)
-        parser.add_argument('--learning_rate', type=float, default=0.0001)
-        return parent_parser
+
+class MyLightningCLI(LightningCLI):
+
+    def before_parse_arguments(self, parser):
+        parser.set_defaults(seed_everything=1234)
```

Comment:
Could we pass the seed directly to the `LightningCLI`?

Reply:
Yes, could be.

Reply:
Then let's do so, I would like to not have to subclass for the seed.

Reply:
Done